lib/types.ts

```diff
@@ -0,0 +1,51 @@
+// Common type aliases for better code readability
+
+import type { Config as JestConfig } from '@jest/types';
+import type * as JestRunner from 'jest-runner';
+
+export type Milliseconds = number;
+export type Path = string;
```

Comment:
```suggestion
export type Path = JestConfig.Path;
```

Reply:
Should I also change `export type Milliseconds = number;` to
```ts
import type { TestResult as JestTestResult } from "@jest/types";
export type Milliseconds = JestTestResult.Milliseconds;
```

Reply:
yeah 👍